work_items/kyc-vcs/requirements-to-accept-a-kyc-vc.md

```diff
@@ -0,0 +1,590 @@
+         "givenName": "Jean"
+      },
+      "birthDate": "1999/12/25",
+      "postalAddress": {
+         "streetAddress": "31 rue Perdu",
+         "country": "FRA",
+         "locality": "La Ville",
+         "region": "Ile-de-France",
+         "postalCode": "75000"
+      },
+      "nationality": "FRA",
+      "accountType": "DepositAccount",
+      "purposeOfAccount": "Deposits",
+      "occupationalCategory": "3123 Construction Supervisors",
+      "sourceOfFunds": "Employment" 
+   },
+   "proof": { "..." }
+}
+```
+
+**Note:** EU and UK AML regulations require both a Photo ID and address verification, so to meet compliance requirements in these jurisdictions the verifiable credential will require the additional evidence outlined in Section 3 and 4 below to be considered valid.
+
+#### Additional Considerations
+
+1. The employment information requested at onboarding (or during EDD) is typically not easily validated. Compliance teams use this information to either investigate an account or validate the account activity. Collecting the information alone does not add significant value, rather if this information was contained within an employment credential that was issued by a qualified issuer, such as Workday or your employer, the information would be more accurate, could be updated or refreshed when employment information changes and able to be trusted and relied on. Therefore, recommend that we remove this field from the KYC credential and focus on enabling a separate employment credential issued by a qualified organization.
+2. Even for non documentary verification (when there is no document expiration date), we can set standard expiry dates within the credential. For example,  compliance standards dictate that KYC should be refreshed every 1-3 years based on risk.
+3. We are recommending that initially we separate out the Identity and Address Verification Evidence from the KYC Verifiable Credential for several reasons:
+ - For simplicity, we may want to start with more simple credentials which will be easier to digest and accept.
+ - KYC information is constant, unless you change your name the information does not change. However, identity documents and account statements have expiration dates and need to be revoked and reissued on a regular basis. Separating them into separate credentials means one can be revoked when it is no longer valid while the other can remain intact.
+ - By separating out the information in these credentials, users can limit the information that they share with others. For example, progressive onboarding is a common practice in the US, some companies may require only the collection of certain information to start transacting, and later request a photo ID or source of funds validation when the transaction volume reaches a certain threshold. This segregation allows the user to share only the information required, or to share more than one credential when more information is needed up front.
+
```

Comment:
```suggestion
4. Standardized responses for fields are preferable to ensure standardization and to allow for easier storage and data formatting. Otherwise, each Verifier will need to map possible data fields to their own internal databases (to be fair this will need to be done anyway, but at least it will be a one time exercise. Please review and provide input on the possible responses for any fields that do. not have a standard schema.
```